src/site/content/en/blog/backdrop-filter/index.md

```diff
@@ -0,0 +1,187 @@
+- The overlaying element will get a new stacking context
+
+{% Aside 'warning' %}
+`backdrop-filter` may harm performance.
+{% endAside %}
+
+
+## Essentials
+
+### Single Filter
+
+```css
+.blur-behind-me {
+  backdrop-filter: blur(.5rem);
+}
+```
+
+<figure class="w-figure">
+  <video controls autoplay loop muted class="w-screenshot">
+    <source src="https://storage.googleapis.com/web-dev-assets/backdrop-filter/backdrop_filter-rgb.mp4" type="video/mp4">
+  </video>
+  <figcaption class="w-figcaption">
+    <a href="https://codepen.io/netsi1964/pen/JqBLPK" target="_blank">CodePen</a>
+  </figcaption>
+</figure>
+
+### Combining Filters
+
+```css
+.brighten-saturate-and-blur-behind-me {
+  backdrop-filter: brightness(150%) saturate(150%) blur(1rem);
+}
+```
+
+<figure class="w-figure">
+  <video controls autoplay loop muted class="w-screenshot">
+    <source src="https://storage.googleapis.com/web-dev-assets/backdrop-filter/backdrop_filter-ambient_blur.mp4" type="video/mp4">
+  </video>
+  <figcaption class="w-figcaption">
+    <a href="https://codepen.io/pepf/pen/GqZkdj" target="_blank">CodePen</a>
+  </figcaption>
+</figure>
+
+
+## Creative Examples
+
+### Overlays
+
+```css
+.modal {
+  backdrop-filter: blur(10px);
+}
+```
+
+<figure class="w-figure">
+  <video controls autoplay loop muted class="w-screenshot">
+    <source src="https://storage.googleapis.com/web-dev-assets/backdrop-filter/backdrop_filter-modal.mp4" type="video/mp4">
+  </video>
+  <figcaption class="w-figcaption">
+    <a href="https://mfreed7.github.io/backdrop-filter-feature/examples/scrollable.html" target="_blank">Demo</a>
+  </figcaption>
+</figure>
+
+### Text Contrast on Dynamic Backgrounds
+
+```css
+.background {
+  filter: invert(1);
+}
+
+.foreground {
+  backdrop-filter: invert(1);
+}
+```
+
+<figure class="w-figure">
+  <video controls autoplay loop muted class="w-screenshot">
+    <source src="https://storage.googleapis.com/web-dev-assets/backdrop-filter/backdrop_filter-invert_color.mp4" type="video/mp4">
+  </video>
+  <figcaption class="w-figcaption">
+    <a href="https://tympanus.net/codrops-playground/huijing/Qqpwg5Iy/editor" target="_blank">Codrops</a>
+  </figcaption>
+</figure>
+
+### Censorship
+
+```css
+input:not(:checked) ~ .censor {
+  backdrop-filter: contrast(4) blur(20px);
+}
+```
+
+<figure class="w-figure">
+  <video controls autoplay loop muted class="w-screenshot">
+    <source src="https://storage.googleapis.com/web-dev-assets/backdrop-filter/backdrop_filter-censorship.mp4" type="video/mp4">
+  </video>
+  <figcaption class="w-figcaption">
+    <a href="https://tympanus.net/codrops-playground/huijing/6cZVcORz/editor" target="_blank">Codrops</a>
+  </figcaption>
+</figure>
+
+
+## Conclusion
+Chrome's excited to release `backdrop-filter` in 76. The [Chromium bug](https://crbug.com/497522) has **560+ stars** on it, clearly marking it as a desired web design capability.
+
+<figure class="w-figure">
```

Comment:
As with the examples above, I think this figure needs a sentence of setup so I know what it's showing me.